tests/guessit_tests.py

```diff
@@ -0,0 +1,116 @@
+# coding=utf-8
+"""Guessit name parser tests."""
+
+from __future__ import unicode_literals
+
+import os
+import unittest
+from io import open
```

Comment:
I've changed this after the import `unicode_literals`, my initial though was that I wasn't loading the file with the correct encoding

I took a look how guessit was loading the yaml file, then I tried the same:
https://github.com/guessit-io/guessit/blob/master/guessit/test/test_yml.py#L6

But now, I already know the real issue: https://github.com/guessit-io/guessit/issues/326

and I'll just drop this part as it's not needed.